config/config.json

```diff
@@ -2,21 +2,21 @@
   "development": {
     "username": "Jessica",
```

Comment:
get this from process.ENV.DB_USERNAME because your teammates have a different username in their own local computer, you also need to turn the extension from .json to .js to achive this